shacl/ActorOrganisatie.ttl

```diff
@@ -46,36 +52,40 @@ recordtypes:ActorOrganisatie
         sh:group actorOrganisatie:identificationGroup ;
         memorix:inTitleAt 1.0 ;
         sh:order    1.0 ;
-        sh:path actorOrganisatie:identifier ;
+        sh:path rico:identifier
     ] ;
     sh:property [
         rdfs:label "Naam"@nl ;
         sh:maxCount 1 ;
         sh:datatype xsd:string ;
         memorix:inTitleAt 3.0 ;
         sh:order    1.0 ;
-        sh:path actorOrganisatie:name ;
+        sh:path rico:name ;
         sh:group actorOrganisatie:identificationGroup ;
     ] ;
     sh:property [
         rdfs:label "Identiek aan (extern)"@nl ;
-        sh:path actorOrganisatie:sameAs ;
+        # Ben zeer terughoudend in het gebruik van owl:sameAs of vergelijkbare concepten: ze verwachten
+        # _exact_ identieke entiteiten, wat het vaak net niet zijn. owl:sameAs kan veel reasoning verstoren.
+        sh:path rico:isFunctionallyEquivalentTo ;
         sh:maxCount 1 ;
         sh:order 3.0 ;
         sh:group actorOrganisatie:identificationGroup ;
         dash:editor memorix:VocabularyEditor ;
         sh:class skos:Concept ;
-        memorix:conceptSchemeIn ( <https://data.rkd.nl/rkdartists> <http://data.bibliotheken.nl/id/dataset/persons> ) ;
```

Comment:
De link https://data.rkd.nl/rkdartists geeft een 404 dus dit lijkt geen "ding" te zijn. Onduidelijk wat dit zou moeten zijn, heb het daarom achterwege gelaten. Of is dit een Nexus ding voor een controlled vocabulary? Wat is dan de correcte URI voor het ConceptScheme? Enig idee?